mdn/yari
# Flaws

Fixes #428
Fixes #532

To test

1. `yarn clean && yarn start`
2. Open http://localhost:3000 and click "All (en-US) flaws"
3. Now, in a new tab, go to some page that has flaws E.g. http://localhost:3000/en-US/docs/Web/API/WebGL_API/By_example/Boilerplate_1
4. Scroll to the bottom and click to show the flaws
5. Go back to the http://localhost:3000/en-US/_flaws
6. In a new terminal type something like `node content build -l en-us -f web/api`
7. Refresh or just focus on the flaws reporting page and see the number of flaws grow.

Note! The default configuration is `BUILD_FLAW_LEVELS="*:warn"`

To test the CLI run something like:

```
node content build -l en-us -f web/html/element --flaw-levels="macros:error"
```

That should die on the first macros flaw detected.
